.github/workflows/approve-snapshots.yml

```diff
@@ -22,22 +22,25 @@ jobs:
         with:
           ref: ${{ steps.getBranchName.outputs.branch }}
 
-      - name: Install pnpm ⚙️
-        uses: pnpm/action-setup@v2
-        with:
-          version: 7.x
-
       - name: Use Node 16 🕹️
         uses: actions/setup-node@v3
         with:
           node-version-file: 'package.json'
-          cache: 'pnpm'
 
-      - name: Cache Cypress binary 📌
+      - name: Install pnpm ⚙️
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.x
```

Comment:
Moving this step after the Node installation step, now that we no longer use the built-in caching feature of `actions/setup-node`.